net-dns/libidn2/libidn2-2.0.4.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="Internationalized Domain Names (IDN) implementation"
+DESCRIPTION="Libidn2 is a free software implementation of IDNA2008, Punycode and TR46. Its \
+purpose is to encode and decode internationalized domain names."
+HOMEPAGE="https://gilab.com/libidn/libidn2"
+COPYRIGHT="2011-2016 Simon Josefsson"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="http://ftp.gnu.org/gnu/libidn/libidn2-$portVersion.tar.gz"
+CHECKSUM_SHA256="644b6b03b285fb0ace02d241d59483d98bc462729d8bb3608d5cad5532f3d2f0"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libidn2$secondaryArchSuffix = $portVersion
+	lib:libidn2$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:gettext$secondaryArchSuffix
+	lib:texinfo$secondaryArchSuffix
+	git
+	perl
```

Comment:
Why are `git` and `perl` here? If these are only used during build, please move them to `BUILD_PREREQUIRES`